content/kitchen/using_test_kitchen.md

```diff
@@ -0,0 +1,165 @@
++++
+title = "Using Test-Kitchen"
+linkTitle = "Using Test-Kitchen"
+
+[menu.kitchen]
+title = "Using Test-Kitchen"
+identifier = "kitchen/using_test_kitchen"
+parent = "kitchen"
+weight = 11
++++
+
+The release of the Chef Infra 19 introduced a few licensing-related changes to the test-kitchen to ensure compatibility with Infra 19. Test-kitchen needs to comply with the licensing changes for chef-19, however, there is no benefit to the community and hence those changes are not pushed up to test-kitchen/test-kitchen. Instead, we forked chef/test-kitchen.
+
+As part of the forking process, test kitchen has been re-branded to Chef Test Kitchen Enterprise, with the version commencing at 1.0.0. Additionally, the repository has been renamed to chef/chef-test-kitchen-enterprise. There are no changes to how we invoke test-kitchen; this update primarily focuses on branding modifications.
+
+## Installation
+
+The RC1 build of chef-test-kitchen-enterprise is primarily available as a Habitat package. The installation can be downloaded using the following command:
+
+```sh
+sudo hab pkg install --binlink --force chef/chef-test-kitchen-enterprise
+```
+
+The hab package is named `chef/chef-test-kitchen-enterprise`. By using the `--binlink` option, you can add the kitchen binary to your current path. Additionally, the `--force` flag allows you to overwrite any existing binaries.
+
+To verify the success of the installation, run the following command:
+
+```sh
+kitchen version
+```
+
+## Supported drivers, provisioners, and verifier
+
+For the RC1 release, only the kitchen-dokken plugin is supported. This allows us to leverage the Dokken driver to create platform containers and use the Dokken provisioner for configuring the Chef Infra Client 19 within those containers. By default, Dokken employs the `chef/chef-hab` Docker image to establish the Chef Infra Client 19 on the container through a Habitat-based methodology. The default verifier is InSpec 6.
+
+### Sample kitchen.yml for Chef Infra Client 19
+
+```sh
+---
+driver:
+  name: dokken
+  privileged: true
```

Comment:
```suggestion
  privileged: true
  chef_version: unstable
```

Reply:
User has to specify chef_version as unstable to pull latest Chef Infra 19 image for RC1 cc @brianLoomis